lib/potluck.rb

```diff
@@ -0,0 +1,26 @@
+require "pry"
+require "./lib/dish"
+
+class Potluck
+  def initialize(date)
+    @date = date
+    # @dishes = {name => category}
+  end
+
+
+  def date
+    @date
+  end
+
+  def dishes
+    @dishes = []
+  end
+
+  def add_dish(dish)
+    # add a new instance of dish
+    name = dish.name
+    category = dish.category
+    dish = Dish.new(name, category)
+  end
```

Comment:
Might be a good place to use `binding.pry`? You don't need to create a new instance of `Dish` here on 23, because you already have an instance being passed in as `dish`.

The next step is to take that instance that is being passed in and shovel it into your `@dishes` array.

Let me know if you'd like to catch up to discuss!